samples/bluetooth/fast_pair/input_device/README.rst

```diff
@@ -84,7 +84,7 @@ See :ref:`ug_bt_fast_pair_provisioning` in the Fast Pair user guide for details.
 .. tip::
    The sample provides TX power in the Bluetooth advertising data.
    There is no need to provide the TX power value during device model registration.
-   The device is using only Bluetooth LE, so you must select **Skip connecting audio profiles (e.g. A2DP or HFP)** option when registering the device.
+   The device is using only Bluetooth LE, so you must select :guilabel:`Skip connecting audio profiles (e.g. A2DP or HFP)` option when registering the device.
```

Comment:
```suggestion
   The device is using only Bluetooth LE, so you must select :guilabel:`Skip connecting audio profiles (for example, A2DP or HFP)` option when registering the device.
```

Reply:
@peknis  This cannot be edited here, it is a gui item that is named as such. It should be modified by the team responsible for the tool in the tool itself. This is out of scope of this PR. However, after this PR will be merged, I'll open a JIRA about this

Reply:
Oh yes, my bad. I did not think through this. Obviously, it is the tool owners' responsibility.